src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -391,6 +391,142 @@ std::set<std::vector<element::Type>> jit_exp_emitter::get_supported_precisions(c
     return {{element::f32}};
 }
 
+/// GELU_ERF ///
+jit_gelu_erf_emitter::jit_gelu_erf_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
+                                           dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                           const std::shared_ptr<ov::Node>& node)
+        : jit_emitter(host, host_isa, node, get_arithmetic_binary_exec_precision(node)) {
+    prepare_table();
+    exp_emitter = std::make_unique<jit_exp_emitter>(h, host_isa, node);
+}
+
+jit_gelu_erf_emitter::jit_gelu_erf_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
+                                           dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                           const ov::element::Type exec_prc) : jit_emitter(host, host_isa, exec_prc) {
+    prepare_table();
+    exp_emitter = std::make_unique<jit_exp_emitter>(h, host_isa, exec_prc);
+}
+
+size_t jit_gelu_erf_emitter::get_inputs_count() const { return 1; }
+
+size_t jit_gelu_erf_emitter::get_aux_vecs_count() const {
+    return std::max<size_t>(exp_emitter->get_aux_vecs_count() + 3, 7);
```

Comment:
Maybe we can save 2 aux_vecs. Here are my thoughts:
1. As vmm_dst is used very late, vmm_aux can reuse vmm_dst.
2. If we delay the computation of t to when we compute -exp(-x*x)*t, then vmm_aux_t can reuse vmm_aux0.

What do you think? If they make sense, I hope they won't take much efferts, otherwise we can do it in separate PR.

Reply:
Thanks for comment, extra auxiliary registers are potential issue in the future. But your suggestions are not clear, can you provide more details? Thanks!

Note, please, source and destination registers can be the same. If not, then value in source register can not be changed. 

Some comments:
1. It's not possible to use destination register if source value should be stored.
2. It doesn't matter what value we should store in auxillary register which are not used by `exp_emitter`: `x / sqrt(2)` or `t`. We need to store this value in auxillary register which are not used `exp_emitter` (`register index >= exp_emitter->get_aux_vecs_count()`).

Reply:
1. Cool! Thanks for the reminder. For sure, jit_gelu_erf_emitter itself also supports shared src and dst. Let do not use vmm_dst.
2. I mean can we move the computation of `t` https://github.com/eshoguli/openvino/blob/es/cpu/jit_gelu_erf/src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp#L451-L456 to right before the computation of `-exp(-x*x)*t` https://github.com/eshoguli/openvino/blob/es/cpu/jit_gelu_erf/src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp#L475 . Then we don't need an extra aux_vec for vmm_aux_t. Since exp is done, and its 4 aux_vecs are free. Perhaps we can reuse vmm_aux3 (Previously I was suggesting vmm_aux0, but it seems would be ocuppied). What do you think?

Reply:
Got it, thanks. If we move `t` calculation after `exp_emitter->emit_code`, then we need to store `x` value , which is stored in `vmm_aux0`. If not, I need more details about your suggestion. Can we do offline? it in separate PR?

P.S. `x = abs(x)`, do you remember removed comment `// vmm_aux0 lost sign but it doesn't matter`? :)

Reply:
You are right! Now I believe all the aux_vecs are necessary. Thanks Edward for the elaboration!